hw/top_darjeeling/sw/autogen/chip/top_darjeeling.rs

```diff
@@ -397,7 +397,7 @@ pub const OTBN_BASE_ADDR: usize = 0x22100000;
 /// memory-mapped registers associated with this peripheral should have an
 /// address between #OTBN_BASE_ADDR and
 /// `OTBN_BASE_ADDR + OTBN_SIZE_BYTES`.
-pub const OTBN_SIZE_BYTES: usize = 0x40000;
+pub const OTBN_SIZE_BYTES: usize = 0x20000;
```

Comment:
This seems pretty big, no? It's 131072 bytes. I guess it's the combined memory of IMEM+DMEM+MMIO? How is it calculated?